app/components/Form/ProjectFormSummary.tsx

```diff
@@ -113,7 +114,14 @@ const ProjectFormSummary: React.FC<Props> = (props) => {
           }}
         />
       )}
-    </>
+      <style jsx>
+        {`
+          div :global(dd) {
+            margin-bottom: 2px;
```

Comment:
could we set this style on the widgets themselves rather than making it global. This would ensure consistency across forms

Reply:
Gave this a go - having trouble with rending this in other components.